File: frontend/src/plugins/simulators/directory/manifest.ts
import { BookOpen } from 'lucide-react'
import type { PluginManifest } from '../../types'

export const manifest: PluginManifest = {
  code: 'directory',
  name: 'plugins.directory.name',
  description: 'plugins.directory.description',
  category: 'simulator',
  icon: BookOpen,
  defaultColor: 'green',
  supportedFormats: ['text'],
  defaultEnabled: true,
  comingSoon: false,
  sortOrder: 1,
  injectTypes: ['directory'],
  playerRoute: null,
  PlayerPage: null,
  EventReceiver: null,
  PreviewPopup: null,
  configSchema: null,
}

File: backend/app/plugins/simulators/directory/manifest.py
from app.plugins.base import PluginCategory, PluginManifest

manifest = PluginManifest(
    code="directory",
    name="Annuaire de Crise",
    name_en="Crisis Directory",
    description="Gestion des contacts et ressources de crise",
    category=PluginCategory.SIMULATOR,
    icon="BookOpen",
    default_color="green",
    supported_formats=["text"],
    default_enabled=True,
    coming_soon=False,
    sort_order=1,
    inject_types=["directory"],
    router_module="app.routers.crisis_contacts",
    router_prefix="/api/crisis-contacts",
)
